src/docker/README.md

```diff
@@ -1,72 +1,69 @@
+## Start up
 
-## Build
+> This command may a while the first time you run it, since it will have to compile the Aptos CLI and an indexer node from source before running several commands against a local testnet.
+> Subsequent calls should be much faster, however, due to Docker's caching mechanism.
 
-```bash
+```sh
 # From Econia repo root
-docker build . \
-    --file src/docker/aptos-cli/Dockerfile \
-    --tag aptos-cli
+docker compose --file src/docker/compose.e2e.yml up --detach
 ```
 
-## Version check
+While the local testnet is running, you can look up on-chain Move resources using the published node REST API port (note that the Aptos faucet API may take longer to start up than the node REST API):
 
-```bash
-docker run aptos-cli
+```sh
+# From Econia repo root
+ECONIA_ADDRESS=$(cat src/docker/chain/accounts/econia.address)
+FAUCET_ADDRESS=$(cat src/docker/chain/accounts/faucet.address)
+aptos account list --account $FAUCET_ADDRESS --url http://localhost:8080
 ```
 
-# Local testnet
-
-## Build
-
-After building the Aptos CLI image locally:
+## Shut down
 
-```bash
+```sh
 # From Econia repo root
-docker build . \
-    --file src/docker/local-testnet/Dockerfile \
-    --tag local-testnet
+docker compose --file src/docker/compose.e2e.yml down
 ```
 
-This command uses plaintext (compromised) private keys to publish Econia and the Econia faucet under the following single-signer vanity address accounts:
+# Helpful Docker commands
 
-```bash
-# From Econia repo root
-ECONIA_ADDRESS=$(cat src/docker/local-testnet/accounts/econia.address)
-FAUCET_ADDRESS=$(cat src/docker/local-testnet/accounts/faucet.address)
-```
+- List all images:
 
-## Serve
+  ```
+  docker images -a
+  ```
 
-Once the image is built, you can serve a local testnet containing the Econia and Econia faucet packages as a background process (via detached container mode).
-This command publishes the local testnet node REST API on port 8080 and the Aptos faucet API on port 8081:
+- Run a shell inside an image:
 
-```bash
-CONTAINER_ID=$(docker run \
-    --detach \
-    --publish 8080:8080 \
-    --publish 8081:8081 \
-    local-testnet)
-```
+  ```
+  docker run -it --entrypoint sh <IMAGE_NAME>
+  ```
 
-While the local testnet is running, you can look up the Econia faucet account using the published node REST API port (note that the Aptos faucet API may take longer to start up than the node REST API):
+- List all containers:
 
-```bash
-aptos account list --account $FAUCET_ADDRESS --url http://localhost:8080
-```
+  ```
+  docker ps -a
+  ```
 
-## Shut down
+- Stop and remove all containers:
 
-To shut down the container:
+  ```
+  docker ps -aq | xargs docker stop | xargs docker rm
+  ```
 
-```bash
-docker stop $CONTAINER_ID
-```
+- Prune all containers, images, and volumes:
+
+  ```
+  docker system prune -af --volumes
+  ```
```

Comment:
Seems weird to have Docker documentation in our own documentation, but I agree that this can be very helpful for someone that isn't familiar to Docker but wants to use our Docker setup. I would add a link to the official Docker documentation either at the end or at the top of this section.

Reply:
Link with explanation added in f1b55ad